pipelines/calibrate.py

```diff
@@ -1,13 +1,16 @@
+# TODO DM-40388: Remove this file, once the new pipeline is vetted.
 # Config override for lsst.pipe.tasks.calibrate.CalibrateTask
-# This config is for the ap_verify datasets that use gaia/panstarrs as their
-# refcat names.
+# Ensure we're using the refcats that are actually in this dataset, regardless
+# of what task defaults are.
 
 # Use gaia for astrometry (phot_g_mean for everything, as that is the broadest
 # band with the most depth).
 
 config.connections.astromRefCat = "gaia_dr2_20200414"
 config.astromRefObjLoader.anyFilterMapsToThis = "phot_g_mean"
-config.astromRefObjLoader.filterMap = {}
 
 # Use panstarrs for photometry (grizy filters).
 config.connections.photoRefCat = "ps1_pv3_3pi_20170110"
+config.photoRefObjLoader.filterMap = {
+    "u": "g",
+    "VR": "g"}
```

Comment:
Should this commit be merged with "Refresh the unit test data repo", or did the tests work fine without these changes?

Reply:
Tests work without these changes.